NuGet.config

```diff
@@ -7,7 +7,6 @@
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from DotNet-msbuild-Trusted -->
-    <add key="darc-pub-DotNet-msbuild-Trusted-8ffc3fe" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-DotNet-msbuild-Trusted-8ffc3fe3/nuget/v3/index.json" />
```

Comment:
@mmitche do you what this is about? Maestro did that

Reply:
It removes darc* feeds it doesn't think it needs.